### PR TITLE
Fix phonon linking for Qt4 on OSX

### PR DESCRIPTION
**Pull request by gillins**
This PR fixes `pyside` linking to phonon. I was able to remove the `-E phonon` part from `pyside`'s `build.sh` and it build and tested fine.

See https://github.com/conda-forge/qt-feedstock/issues/34

I was unable to test `pyqt` due to https://github.com/conda-forge/pyqt-feedstock/issues/20

### CODE REVIEW

**Comment by jakirkham** [comment]
@ccordoba12, could you please look at this one?

**Comment by ccordoba12** [comment]
The patch seems fine (I mean, if @gillins says it work for him, I trust him).

This requires a bump in build number (although it'd be better to just bump the number for `osx`).

**Comment by jakirkham** [comment]
So there is one, but it is under the `test` label. Probably better to bump in this case so we can distinguish the two.

**Comment by gillins** [comment]
> Probably better to bump in this case so we can distinguish the two

done

**Comment by gillins** [comment]
Tested `pyqt4` version 4 with https://github.com/conda-forge/pyqt-feedstock/pull/21. All ok.

**Comment by ccordoba12** [comment]
Great work @gillins! Merging then :-)

**Comment by jschueller** [comment]
great news!

**Comment by jschueller** [comment]
shouldn't we bump the release number ?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,9 +36,13 @@ source:
...
     # https://fami.codefreak.ru/gitlab/peter/qt4/commit/45e8f4eef3923e03c6939d0c17170980685857ef.diff
     - vs2015_int_fix.patch  # [win and py >= 35]
+    # see https://bugreports.qt.io/browse/QTBUG-37209
+    # fix phonon exports so they can be linked to
+    - phonon_osx_exports.patch  # [osx]
 
 build:
-  number: 7
+  number: 7  # [linux or win]
+  number: 8  # [osx]
```
Comment (gillins) [inline recipe/meta.yaml]: @jschueller It was bumped to 8 here

Comment (jschueller) [inline recipe/meta.yaml]: ok, thanks

**Pull request merged**